tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.cpp

```diff
@@ -39,7 +39,9 @@
 namespace tcf {
 
     void WorkOrderDataHandler::Unpack(EnclaveData* enclaveData,
-                                      const JSON_Object* object) {
+        const JSON_Object* object, EnclaveType enclave_type,
+        ByteArray data_encryption_key_from_pre_processing) {
+
```

Comment:
Create an overloaded version of `WorkOrderDataHandler::Unpack()` as a first step. Later on we can subclass `WorkOrderDataHandler`

Reply:
Done